mne/io/fil/tests/test_fil.py

```diff
@@ -140,3 +143,21 @@ def test_fil_all():
     _fil_megmag(raw, mat)
     _fil_stim(raw, mat)
     _fil_sensorpos(raw, mat)
+
+
+@testing.requires_testing_data
+def test_fil_no_positions(tmp_path):
+    """Test FIL reader in cases where a position file is missing."""
+    test_path = tmp_path / "FIL"
+    shutil.copytree(fil_path, test_path)
+
+    posname = test_path / "sub-noise_ses-001_task-noise220622_run-001_positions.tsv"
+    binname = test_path / "sub-noise_ses-001_task-noise220622_run-001_meg.bin"
+
+    remove(posname)
+
+    with pytest.warns(RuntimeWarning, match="No sensor positions*"):
+        raw = read_raw_fil(binname)
+        chs = raw.info["chs"]
+        locs = array([ch["loc"][:] for ch in chs])
+        assert isnan(locs).all()
```

Comment:
Limiting the scope of the context manager helps make it clearer where the warning arises
```suggestion
    with pytest.warns(RuntimeWarning, match="No sensor positions*"):
        raw = read_raw_fil(binname)
    chs = raw.info["chs"]
    locs = array([ch["loc"][:] for ch in chs])
    assert isnan(locs).all()
```

Reply:
I used the wildcard so I didn't need to write out the enitrety of "No sensor positions could be found". So I think we might be failling a few unit tests.

Reply:
Ahh it's because it's singular `sensor position` not `positions`. Will fix. (Keep in mind it's not a glob-style wildcard but rather a regular expression here. But maybe you meant `s*` i.e. on or more s's?)